Source/Core/LMCConst.hh

```diff
@@ -28,33 +28,33 @@ public:
 
     constexpr static double Q()
     {
-        return 1.60217653E-19;
+        return 1.602176634E-19;  // updated from https://pdg.lbl.gov/2021
     } //!< elementary charge  in C(>0)
 
     constexpr static double Hbar()
     {
-        return 1.05457168E-34;
+        return 1.054571817E-34; // updated from https://pdg.lbl.gov/2021
     }//!< hbar in J s-1
 
     constexpr static double HbarC_eV()
     {
-        return 197.326968E-9;
+    	return 197.3269804; // updated from https://pdg.lbl.gov/2021
     }//!<hbar c in m eV.
 
     constexpr static double kB()
     {
-        return 1.3806505E-23;
+        return 1.380649E-23; // updated from https://pdg.lbl.gov/2021
     }//!< Boltzmann constant J/K
 
     constexpr static double kB_eV()
     {
-        return 8.617343E-5;
+        return 8.617333262E-5; // updated from https://pdg.lbl.gov/2021
     }//!< Boltzmann constant eV/K
 
     //EM coupling constants
     constexpr static double EpsNull()
     {
-        return 8.854187817E-12;
+        return 8.8541878128E-12; // updated from https://pdg.lbl.gov/2021
     } //!< epsilon0, Constant of Newtons force.
```

Comment:
this is not "Constant of Newton force." but "permittivity of free space" or "electric constant" there are units as well. They are "F / m" (Farad per meter)